underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -300,25 +298,13 @@ public OutputStream create(String path, CreateOptions options) throws IOExceptio
 
   @Override
   public OutputStream createDirect(String path, CreateOptions options) throws IOException {
-    IOException te = null;
     FileSystem hdfs = getFs();
-    RetryPolicy retryPolicy = new CountingRetry(MAX_TRY);
-    while (retryPolicy.attempt()) {
-      try {
-        // TODO(chaomin): support creating HDFS files with specified block size and replication.
-        OutputStream outputStream = new HdfsUnderFileOutputStream(
```

Comment:
@apc999 @yuzhu  Do you have any context why the retry was added?

Reply:
@jja725 i noticed this part too and wondering why is it not possible to maintain the retry logic?

Reply:
It's possible. there's some conversation about whether we want to keep retry for such external call. I just remove them for consistency(most of our ufs call doesn't have retry) and simplicity